src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -1170,6 +1188,7 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
                             })
                             .collect(),
                     ),
+                    security_context: container_security_context.clone(),
```

Comment:
Can we put these changes behind a LD flag? In case something goes wrong during the release window (e.g., imagine things break only for pods using SSH tunnels, like Paul mentioned above), it'd be nice to be able to just flip the switch and get back to a healthy state.

Reply:
Sure, should be straight forward.